src/scripts/main.js

```diff
@@ -1 +1,12 @@
-'use strict';
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const link = e.target.closest('a');
```

Comment:
Using `e.target.closest('a')` is a good approach for event delegation. However, ensure that the structure of your HTML guarantees that the closest 'a' element is always the correct one you intend to target.